### ISSUE TITLE
SDL_mixer module cannot be compiled

### ISSUE THREAD

**Issue by AsciiWolf** [flathub/shared-modules#18]
I am getting this error when trying to use the _SDL_mixer-1.2.12.json_ module:

```
./configure: line 13698: syntax error near unexpected token `$SMPEG_VERSION,'
./configure: line 13698: `    AM_PATH_SMPEG($SMPEG_VERSION, have_smpeg=yes, have_smpeg=no)'
```

**Comment by AsciiWolf** [comment]
Ah, it was missing the _smpeg_ module. It would be great if the module could be included in the _SDL_mixer-1.2.12.json_ file.

**Comment by AsciiWolf** [comment]
Fixed in a28ded7d85cf310f67e68ccc1d1d003e17463abb.
